client/deposits/instant-deposits/modal.js

```diff
@@ -14,19 +14,22 @@ import { formatCurrency } from 'utils/currency';
 import './style.scss';
 
 const InstantDepositModal = ( {
-	amount,
-	fee,
-	net,
+	instantBalance: { amount, fee, net, fee_percentage: percentage },
 	onClose,
 	onSubmit,
 	inProgress,
 } ) => {
 	const learnMoreHref = 'https://docs.woocommerce.com/document/payments/';
+	const feePercentage = `${ percentage }%`;
 	const description = createInterpolateElement(
+		/* translators: %s - amount representing the fee percentage */
```

Comment:
I think there can be only one `/* translators */` comment per string, so you'll have to combine this and the next into the same line.

Reply:
Updated in https://github.com/Automattic/woocommerce-payments/pull/1673/commits/a9112987157bfa8f0941636c970cf04024e627aa